x/protorev/keeper/keeper_test.go

```diff
@@ -846,16 +850,64 @@ func (suite *KeeperTestSuite) setUpPools() {
 			},
 			scalingFactors: []uint64{1, 1},
 		},
+		{ // Pool 46 - Used for epoch_hook GetHighestLiquidityPool testing
+			initialLiquidity: sdk.NewCoins(
+				sdk.NewCoin("epochOne", sdk.NewInt(1000)),
+				sdk.NewCoin("uosmo", sdk.NewInt(1000)),
+			),
+			poolParams: stableswap.PoolParams{
+				SwapFee: sdk.NewDecWithPrec(1, 4),
+				ExitFee: sdk.NewDecWithPrec(0, 2),
+			},
+			scalingFactors: []uint64{1, 1},
+		},
+		{ // Pool 47 - Used for epoch_hook GetHighestLiquidityPool testing
+			initialLiquidity: sdk.NewCoins(
+				sdk.NewCoin("epochOne", sdk.NewInt(1000)),
+				sdk.NewCoin("uosmo", sdk.NewInt(2000)),
+			),
+			poolParams: stableswap.PoolParams{
+				SwapFee: sdk.NewDecWithPrec(1, 4),
+				ExitFee: sdk.NewDecWithPrec(0, 2),
+			},
+			scalingFactors: []uint64{1, 1},
+		},
+		{ // Pool 48 - Used for epoch_hook GetHighestLiquidityPool testing
+			initialLiquidity: sdk.NewCoins(
+				sdk.NewCoin("epochTwo", sdk.NewInt(1000)),
+				sdk.NewCoin("uosmo", sdk.NewInt(1000)),
+			),
+			poolParams: stableswap.PoolParams{
+				SwapFee: sdk.NewDecWithPrec(1, 4),
+				ExitFee: sdk.NewDecWithPrec(0, 2),
+			},
+			scalingFactors: []uint64{1, 1},
+		},
 	}
 
 	for _, pool := range suite.stableSwapPools {
 		suite.createStableswapPool(pool.initialLiquidity, pool.poolParams, pool.scalingFactors)
 	}
 
+	// Create a concentrated liquidity pool for epoch_hook testing
+	suite.createConcentratedLiquiditiyPool()
+
 	// Set all of the pool info into the stores
 	suite.App.ProtoRevKeeper.UpdatePools(suite.Ctx)
 }
 
+func (suite *KeeperTestSuite) createConcentratedLiquiditiyPool() {
+	// Create a concentrated liquidity pool for epoch_hook testing
+	clPoolID, err := suite.App.PoolManagerKeeper.CreatePool(suite.Ctx, clmodel.NewMsgCreateConcentratedPool(suite.TestAccs[0], "epochTwo", "uosmo", 1, sdk.NewInt(-1), sdk.ZeroDec()))
+	suite.Require().NoError(err)
+
+	clPool, err := suite.App.PoolManagerKeeper.GetPool(suite.Ctx, clPoolID)
+	suite.Require().NoError(err)
+
+	err = suite.App.BankKeeper.SendCoins(suite.Ctx, suite.TestAccs[0], clPool.GetAddress(), sdk.NewCoins(sdk.NewCoin("epochTwo", sdk.NewInt(1000)), sdk.NewCoin("uosmo", sdk.NewInt(2000))))
+	suite.Require().NoError(err)
+}
```

Comment:
`PrepareConcentratedPoolWithCoins` already exists in apptesting > concentrated_liquidity.go

Reply:
Changed to use this already existing function here: https://github.com/osmosis-labs/osmosis/pull/4946/commits/526eaeede97d9f8add9cdb6b6514e272f8cd09a7